spec/system/provider_interface/provider_applications_filter_spec.rb

```diff
@@ -194,14 +194,14 @@ def and_teacher_degree_apprenticeship_filter_is_not_visible
     expect(page).to have_content('Filter')
     expect(page).to have_no_content('Course type')
     expect(page).to have_no_content('Postgraduate courses')
-    expect(page).to have_no_content('Teaching degree apprenticeship (TDA) courses')
+    expect(page).to have_no_content(I18n.t('provider_interface.filters.course_type.teacher_degree_apprenticeship'))
```

Comment:
My feeling is that we use locales in the files that render what we se on the screen, but in tests we ensure those locales match our expectations. So I would rather this (and the others below) stay as they are.

Reply:
Makes sense. Done